ubr/conf.py

```diff
@@ -38,12 +38,17 @@
 _handler.setLevel(logging.DEBUG)
 _handler.setFormatter(logging.Formatter('%(levelname)s - %(asctime)s - %(message)s'))
 
+_filehandler = logging.FileHandler('ubr.log')
```

Comment:
ubr is run as root so it should need to be created. logrotation should be a thing